broken link fixed w/signoff (#9960)
meshery/meshery: docs/pages/guides/configuration-management/configuration-management.md

You can also import these filters manually to your provider from the [wasm-filters](https://github.com/layer5io/wasm-filters) repo.

> Meshery's sample application [ImageHub](/docs/pages/guides/infrastructure-management/deploying-sample-apps.md) will let you test out configuring these filters out-of-the-box.

Reviewer: ```suggestion
Meshery's sample application [ImageHub]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
```

@prathyushmadhu please accept and use this change.

Don't include filename extension.
Don't include "/docs".
Do use `{{ site.baseurl }}`.

Author: @leecalcote I have made the required changes and have commited to the same PR.

> Meshery's sample application [ImageHub]({{ site.baseurl }}/pages/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.

Reviewer: ```suggestion
Meshery's sample application [ImageHub]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
```

### Managing Applications Through Meshery UI

> Meshery has a set of [sample applications]({{ site.baseurl }}/pages/guides/infrastructure-management/deploying-sample-apps) which you can use to quickly test out your deployment.

Reviewer: ```suggestion
Meshery has a set of [sample applications]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) which you can use to quickly test out your deployment.
```

- Helm charts
- Docker Compose Apps

> You can also run the same application on multiple connected kubernetes clusters and compare the performance. See [Performance Management with Meshery]({{ site.baseurl }}/pages/guides/performance-management/performance-management) for more details.

Reviewer: ```suggestion
You can also run the same application on multiple connected kubernetes clusters and compare the performance. See [Performance Management with Meshery]({{ site.baseurl }}/guides/performance-management/performance-management) for more details.
```
